### PR TITLE
Minor Fixes

### PR DESCRIPTION
**Pull request by tomirons**

### CODE REVIEW

File: src/Models/Discussion.php [hunk 1]
```diff
@@ -34,18 +33,33 @@ class Discussion extends Model
...
-    public function post()
+    /**
+     * Get the first post of the discussion.
+     *
+     * @return mixed
+     */
+    public function firstPost()
     {
-        return $this->posts->first()->with('user')->first();
+        return $this->posts()->orderBy('created_at')->with('user')->first();
```
Comment (Dylan-DPC) [inline src/Models/Discussion.php]: can  use `oldest()` works better in terms of readability

Comment (tomirons) [inline src/Models/Discussion.php]: This has been done. 😄

**Comment by Riari** [comment]
Question: why add the standard timestamp columns to the $date property in models? If the $timestamps property is set to `true` (which it will be by default) then they'll already be cast to Carbon objects automatically.

**Comment by tomirons** [comment]
I didn't know that was a thing, I've always added them.

**Comment by Riari** [comment]
Dunno, it's been there for as long as I can remember! It may have been added with the first release of Laravel 5 though. Not sure, but we can safely leave out the $dates attribute if we're only using the standard timestamps.

**Pull request merged**